File: src/targets/python/python3/fixtures/jsonObj-multiline.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "{\n  \"foo\": \"bar\"\n}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/node/axios/fixtures/jsonObj-multiline.js
const axios = require('axios').default;

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {'content-type': 'application/json'},
  data: {foo: 'bar'}
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}